README.md

```diff
@@ -1,87 +1,60 @@
+* `~adjust_ros_timestamp` (`bool`, default: `true`): Use sensor timestamp instead of ROS timestamp to mitigate USB communication issues.
+* `~map_frame_id` (`string`, default: `map`): The map frame id of the Vectornav device. For example, `map`.
 
-Run:
+* `~linear_accel_covariance` (`double[9]`, default: `[0.01, 0, 0, 0, 0.01, 0, 0, 0, 0.01]`): The covariance of the linear acceleration.
+* `~angular_vel_covariance` (`double[9]`, default: `[0.01, 0, 0, 0, 0.01, 0, 0, 0, 0.01]`): The covariance of the angular velocity.
+* `~orientation_covariance` (`double[9]`, default: `[0.01, 0, 0, 0, 0.01, 0, 0, 0, 0.01]`): The covariance of the orientation.
 
-```bash
-(Terminal 1) $ roscore
-(Terminal 2) $ roslaunch vectornav vectornav.launch
-(Terminal 3) $ rostopic list
-(Terminal 3) $ rostopic echo /vectornav/IMU
-(Terminal #) $ ctrl+c to quit
-```
+* `~rotation_reference_frame` (`double[9]`, default: `[1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0, 0, 1.0]`): The rotation reference frame of the Vectornav device. For example, `[1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0, 0, 1.0]` for NED frame.
 
+### Topics
 
-Overview 
---------
+The following topics are published by the `vnpub` node:
+* `~data` (`sensor_msgs/Imu`): The IMU data with standard ROS messages.
+* `~mag` (`sensor_msgs/MagneticField`): The magnetic field data with standard ROS messages.
+* `~gps` (`sensor_msgs/NavSatFix`): The GPS data with standard ROS messages.
+* `~odom` (`nav_msgs/Odometry`): The odometry data with standard ROS messages.
+* `~temperature` (`sensor_msgs/Temperature`): The temperature data with standard ROS messages.
+* `~atm_pressure` (`sensor_msgs/FluidPressure`): The atmospheric pressure data with standard ROS messages.
+* `~ins_status` (`vectornav_msgs/InsStatus`): The INS status data with custom ROS messages.
 
-#### vnpub node
+### Services
 
-This node provides a ROS interface for a vectornav device. It can be configured
-via ROS parameters and publishes sensor data via ROS topics.
+The following services are provided by the `vnpub` node:
+* `~set_acc_bias` (`vectornav_msgs/SetAccBias`): Set the accelerometer bias to `[0.0, 0.0, 9.81]` with mean of samples taken during `set_acc_bias_seconds` seconds.
+* `~reset_acc_bias` (`std_srvs/Empty`): Reset the accelerometer bias to zero.
 
+> **_NOTE:_** The services are only available if `acc_bias_enable` is set to `true`.
 
-#### vectornav.launch
+## Code of Conduct
+This project has adopted the [Robotnik Code of Conduct](https://to.do/code_of_conduct_faq). For more information see the [Code of Conduct FAQ](https://to.do/code_of_conduct_faq) or contact [opencode@robotnik.es](opencode@robotnik.es) with any additional questions or comments.
 
-This launch file contains the default parameters for connecting a device to ROS.
-You will problaby want to copy it into your own project and modify as required. 
+## License
+Copyright (c) 2023, Robotnik Automation S.L.L. All rights reserved.
 
+Licensed under the [BSD 2-Clause](./LICENSE) License.
```

Comment:
The LICENSE doc does not exist

Reply:
Done with commit 5f58318ff52f2350696aea47509925678c183006